snapshots/js/global.js

```diff
@@ -0,0 +1,66 @@
+window.GUIDE_VERSIONS = {
+    current: null,
+    latest: null,
+    available: [],
+
+    onReady: function(callback) {
+        if (this._ready) {
+            callback(this);
+        } else {
+            this._callbacks.push(callback);
+        }
+    },
+
+    urlFor: function(version) {
+        return window.location.href.replace(this.current, version);
+    },
+
+    _callbacks: [],
+
+    _setData: function(available) {
+        function parseVersion(version) {
+            var match = version.match(/^v(\d+)\.(\d+)\.(\d+)$/);
+            if (match) {
+                return {
+                    major: match[1],
+                    minor: match[2],
+                    patch: match[3]
+                };
+            }
+        }
+
+        function compareVersions(a, b) {
+            a = parseVersion(a);
+            b = parseVersion(b);
+
+            if (a.major !== b.major) { return a.major - b.major; }
+            if (a.minor !== b.minor) { return a.minor - b.minor; }
+            return a.patch - b.patch;
+        }
+
+        // Sort versions and make latest appear first
+        this.available = available.sort(compareVersions).reverse();
+
+        // @TODO: have to work through how this gets populated when we're using a global js include on an older version of Ember
+        this.current = 'CURRENT_REVISION_WILL_APPEAR_HERE_WHEN_DEPLOYED';
```

Comment:
Our current `window.GUIDE_VERSIONS` hard-codes current.  Seems like we'd need to pull it from the URL of the page we're looking at instead of we're going to be using this globally ...